### ISSUE TITLE
Plan for Swift 2.3 support

### ISSUE THREAD

**Issue by phubner** [danthorpe/YapDatabaseExtensions#88]
Is there a plan to support swift 2.3?

**Comment by danthorpe** [comment]
Hi @phubner - yes, absolutely planned. It's next on the list after ValueCoding is done.

**Comment by phubner** [comment]
hey @danthorpe, I noticed that ValueCoding has a `swift/2.3` branch. I assumed that is where they/you are keeping the 2.3 compatible version? Is that not a release ready branch?

**Comment by danthorpe** [comment]
Oh, yes, but I think now that Xcode 8 is GM, then I'll do a proper release of ValueCoding - get a tagged version to depend on for _YapDatabaseExtensions_. Will try to do this over the next few nights.

Sorry it's a bit late - I've got quite a few open source projects, Swift 3 has been tough!

**Comment by phubner** [comment]
Ha! No worries, I'm not looking forward to switching over yet. But know I am going to have to face that soon enough.

**Comment by danthorpe** [comment]
Yeah, I think my plan for _YapDatabaseExtensions_, for Swift 3.0, will be a bit of a stripping back to refactor some of the API. I think most people only use the "functional" API - so will keep that, but might get rid of the more fancy stuff. This was mostly just me learning more Swift stuff anyway.
